config/permissions.yml

```diff
@@ -727,6 +727,14 @@ field_customer_accessibility_explanation_edit:
     expose: true
     loginRequired: true
     parent: area_customer_settings
+feature_customer_support_contact_administration:
+    description: |-
+        Enables the option to create/edit/delete supportContact entities.
+        A customer-admin can post support contact informattions visible to everyone who is in that customer context.
```

Comment:
typo informattions

Reply:
done in 3f5993c305cc8ddfc09127e04b599dc0f70cf744

Reply:
Almost, information is the pural of information :-)

Reply:
next attempt :) 7aaf2819d5ee82c884f900b8e3fb4dd647cee588